### PR TITLE
Feature/el vagrant

### PR DESCRIPTION
**Pull request by eriklundberg**

### CODE REVIEW

File: db/migrations/011_create_intake_table.sql [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+    CONSTRAINT "pollster_results_intake_Q4c_check1" CHECK (("Q4c" >= 0)),
+    CONSTRAINT "pollster_results_intake_Q4d1_check1" CHECK (("Q4d1" >= 0)),
+    CONSTRAINT "pollster_results_intake_Q4d2_check1" CHECK (("Q4d2" >= 0)),
+    CONSTRAINT "pollster_results_intake_Q4d_check1" CHECK (("Q4d" >= 0)),
+    CONSTRAINT "pollster_results_intake_Q4e1_check1" CHECK (("Q4e1" >= 0)),
+    CONSTRAINT "pollster_results_intake_Q4e2_check1" CHECK (("Q4e2" >= 0))
+);
+
+
+ALTER TABLE public.pollster_results_intake OWNER TO admin;
```
Comment (hrpt-se) [inline db/migrations/011_create_intake_table.sql]: Is it "admin" that owns the table or "epiwork"?

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+chmod 600 ~/.pgpass
+
+pip install -r /vagrant/requirements.txt
+
+
+sudo -u postgres psql <<EOF
+  DROP DATABASE IF EXISTS epiwork;
+  DROP USER IF EXISTS admin;
+  CREATE USER admin WITH PASSWORD 'admin' SUPERUSER;
+  CREATE DATABASE epiwork WITH OWNER = admin;
```
Comment (hrpt-se) [inline install.sh]: Why the change of "owner" from epiwork to master?

File: db/migrations/011_create_intake_table.sql (hunk 1 above)
Comment (eriklundberg) [inline db/migrations/011_create_intake_table.sql]: I think you're looking at an old commit. In the latest version the owner should be "epiwork".
